addons/point_of_sale/static/tests/tours/helpers/ProductScreenTourMethods.js

```diff
@@ -154,6 +154,27 @@ odoo.define('point_of_sale.tour.ProductScreenTourMethods', function (require) {
                 },
             ];
         }
+        clickQuotationButton() {
+            return [
+                {
+                    content: 'click quotation button',
+                    trigger: '.o_sale_order_button',
+                }
+            ];
+        }
+
+        selectFirstOrder() {
+            return [
+                {
+                    content: `select order`,
+                    trigger: `.order-row .col.name:first`,
+                },
+                {
+                    content: `click on select the order`,
+                    trigger: `.selection-item:contains('Settle the order')`,
+                }
+            ];
+        }
```

Comment:
This should be inside the `pos_sale` module. Check the ProductScreenTourMethods of pos_restaurant to see how to extends